src/components/Notes/Notes.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import NotesList from "./NotesList";
+import Search from "./Search";
+import UpdateNoteDialog from "./UpdateNoteDialog";
+const Notes = () => {
+  const [notes, setNotes] = useState([]);
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const [currentNote, setCurrentNote] = useState(null);
+
+  const fetchNotes = async () => {
+    try {
+      const response = await axios.get("http://localhost:3000/notes");
+      setNotes(response.data);
+    } catch (error) {
+      console.error("Error fetching notes:", error);
+    }
+  };
+
+  const handleSelectNote = (id) => {
+    const note = notes.find((note) => note._id === id);
+    setCurrentNote(note);
+    setDialogOpen(true);
+  };
+
+  const handleSaveNote = async (id, title, content) => {
+    try {
+      await axios.patch(`http://localhost:3000/notes/${id}`, {
+        title,
+        content,
+      });
```

Comment:
```suggestion
    const handleSaveNote = async (id, data) => {
    try {
      await axios.patch(`http://localhost:3000/notes/${id}`, data);
```

group all request data in single parameter